CONTRIBUTING.md

```diff
@@ -0,0 +1,172 @@
+```
+Now you are free to play with your fork (for additional information, you can visit the
+[Github fork help page](https://help.github.com/en/articles/fork-a-repo)).
+
+> Note: you do not have to re-do the setup above every time.
+> Instead, in the future, you need to update the `development` branch
+> on your fork with
+> ```
+> git checkout development
+> git pull development
+> ```
+
+Make sure you are on the `development` branch with
+```
+git checkout development
+```
+in the AMReX directory.
+
+Create a branch `<branch_name>` (the branch name should reflect the piece
+of code you want to add, like `high_order_interpolation`) with
+```
+git checkout -b <branch_name>
+```
+and do the coding you want.
+Add the files you work on to the git staging area with
+```
+git add <file_I_created> <and_file_I_modified>
+```
+### Commit & push your changes
+
+Periodically commit your changes with
+```
+git commit -m "This is a 50-char description to explain my work"
+```
+
+The commit message (between quotation marks) is super important in order to
+follow the developments and identify bugs.
+
+For the moment, commits are on your local repo only. You can push them to
+your fork with
+```
+git push -u origin <branch_name>
+```
+
+If you want to synchronize your branch with the `development` branch (this is useful
+when `development` is being modified while you are working on
+`<branch_name>`), you can use
+```
+git pull upstream development
+```
+and fix any conflicts that may occur.
+
+### Submit a Pull Request
+
+A Pull Request is the way to efficiently visualize the changes you made
+and to propose your new feature/improvement/fix to the AMReX project.
+Right after you push changes, a banner should appear on the Github page of
+your fork, with your `<branch_name>`.
+- Click on the `compare & pull request` button to prepare your PR.
+- It is time to communicate your changes: write a title and a description for
+your PR. People who review your PR are happy to know
+  * what feature/fix you propose, and why
+  * how you made it (created a new class than inherits from...)
+  * and anything relevant to your PR (performance tests, images, *etc.*)
+- Press `Create pull request`. Now you can navigate through your PR, which
+highlights the changes you made.
+
+Please DO NOT write large Pull Requests, as they are very difficult and
+time-consuming to review. As much as possible, split them into small,
+targeted PRs.
+For example, if find typos in the documentation open a pull request that only fixes typos.
+If you want to fix a bug, make a small pull request that only fixes a bug.
+If you want to implement a large feature, write helper functionality first, test it and submit those as a first pull request.
+If you want to implement a feature and are not too sure how to split it, just open an issue about your plans and ping other AMReX developers on it to chime in.
+
+Even before your work is ready to merge, it can be convenient to create a PR
```

Comment:
Agreed with @zingale that we should recommend Draft Pull Requests here as an alternative option.

Reply:
I agree.